code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -716,6 +826,13 @@
 	ammo_type = /obj/item/ammo_casing/a12g/beanbag
 	matter = list(MAT_STEEL = 710) //beanbags x2 + 350 metal
 
+/obj/item/ammo_magazine/clip/c12g/bb
+	name = "ammo clip (12g BB)"
+	icon_state = "12gclipshell"
+	desc = "A replica color-coded metal clip for holding and quickly loading shotgun shells. This one is loaded with airsoft BBs."
+	ammo_type = /obj/item/ammo_casing/a12g/pellet
```

Comment:
Shouldn't this be `/obj/item/ammo_casing/a12g/bb`?

Reply:
I *believe* I fixed it?